### DISCUSSION TITLE
Is there any chance the package name can be renamed to "openstitching" or something? "stitching" conflicts with our codebase, and python has no good way of handling this.

### DISCUSSION THREAD

**Discussion by JustinMBrown** [OpenStitching/stitching#252] [General]
I get this is a big ask, but I'm sure there's a fair amount of codebases who have an internal module called stitching that they can't change. I've tried many work arounds, but at this point the cleanest way to solve it is to just copy and paste the entire open stitching codebase into ours which obviously sucks.

Has anyone else run into this?

**Comment by lukasalexanderweber** [comment]
> Is there any chance the package name can be renamed to "openstitching" or something?

no, sorry

> Has anyone else run into this?

maybe someone has a good idea for this. [git submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodules) might be a good way to seperate a fork of this repo with an adjusted package name from your codebase.
